### DISCUSSION TITLE
Add Passkey into Laravel Backpack

### DISCUSSION THREAD

**Discussion by kiddtang** [Laravel-Backpack/community-forum#1210] [Show and Tell]
A Step-by-Step Tutorial: Adding Passwordless Login to Your Admin Panel

[https://medium.com/@dogcomp/boost-laravel-backpack-security-with-webauthn-passkeys-b8ed4fb8052c](https://medium.com/@dogcomp/boost-laravel-backpack-security-with-webauthn-passkeys-b8ed4fb8052c)
You need to be a Medium member to read this article. Thanks for your support.

**Comment by jcastroa87** [comment]
Hello @kiddtang thanks for sharing this with community.

Cheers.

**Reply by emjayess** [reply]
Closed-source content for an open-source project isn't exactly sharing with the community.

**Reply by tabacitu** [reply]
Fair enough! I'm pretty sure @kiddtang only posted it on Medium because it was easiest to him, and wanted to participate in [this months' "Write and Win" promo](https://backpackforlaravel.com/articles/news/progress-report-september-2024). THERE'S STILL TIME BTW, a few more days where you can write an article and win a license!

If you put it on HackMD @kiddtang ... I'd love to take a look and post it on the Backpack Blog... 👀 That way the entire community can see it. We can link to your website/medium/youtube/whatever.

**Reply by kiddtang** [reply]
@tabacitu I've done that. Please check your email 😊

**Reply by kiddtang** [reply]
thanks for publishing it 😊
https://backpackforlaravel.com/articles/tutorials/how-to-add-webauthn-passkeys-to-backpack-for-laravel-admin-panel
